wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/functions.php

```diff
@@ -203,3 +230,104 @@ function wporg_ov_customizer( $wp_customize ) {
 	) );
 }
 add_action( 'customize_register', __NAMESPACE__ . '\wporg_ov_customizer' );
+
+/*
+	=====================================
+	Openverse standalone site redirection
+	=====================================
+ */
+
+/**
+ * This is the URL at which the standalone Openverse site is hosted. When
+ * redirect is enabled (see setting `ov_is_redirect_enabled`), the theme
+ * redirects all incoming requests to the right URL on this domain.
+ *
+ * Note: Do not put a trailing slash '/' in this URL, as it will cause problems.
```

Comment:
I don't remember what it was exactly, but iirc it was that paths already start with `/` so having a trailing slash here would cause them to appear twice.

```
https://openverse.wordpress.net/ + /path
```

Reply:
Added the comment in e390c58.